File: 프로그래머스/0/120892. 암호 해독/암호 해독.py
def solution(cipher: str, code: int) -> str:
    answer = ""
    print(len(cipher))

    if code == 1 or len(cipher) == 1:
        answer = cipher
    else:
        for i in range(code, len(cipher) + 1, code):
            answer += cipher[i - 1]

    return answer

File: 프로그래머스/0/120892. 암호 해독/암호 해독.js
function solution(cipher, code) {
    let answer = "";

    if (code === 1 || cipher.length === 1) {
        answer = cipher;
    } else {
        for (let i = code; i <= cipher.length; i += code) {
            answer += cipher[i - 1];
        }
    }

    return answer;
}